lib/soap-connector.js

```diff
@@ -223,7 +224,7 @@ SOAPConnector.prototype.jsonToXML = function (method, json) {
   if (typeof method === 'string') {
     var m = findMethod(this.client, method);
     if(!m) {
-      throw new Error('Method not found in WSDL port types: ' + m);
+      throw new Error(g.f('Method not found in {{WSDL}} port types: %s', m));
```

Comment:
You have to run slt-globalize -e again for intl/en/messages.json to get updated with the brackets change.